source/includes/fact-query-converter-generic.rst

```diff
@@ -0,0 +1,12 @@
+- Consider SQL complexity when using query converter. The AI 
+  model cannot convert highly complex SQL syntax. If your SQL 
+  code isn't converting accurately, refactor it and try again.
+
+- Query converter utilizes metadata from your relational and MongoDB 
+  connections for syntax conversion. For optimal results, use queries 
+  aligned with your project's relational database schema.
```

Comment:
> For optimal results, use queries aligned with your project's relational database schema.

I'm not sure what this means. Is this saying to specify queries that use table / field names in my relational schema?

Reply:
Thanks, I think this is much clearer.